### ISSUE TITLE
gutter: "Read annotations row X" aria-label doesn't contain the type of annotation

### ISSUE THREAD

**Issue by avinashbot** [ajaxorg/ace#5499]
### Describe the bug

In the gutter, when an annotation icon is present, the aria-label just states "Read annotations row X", but doesn't provide any information to assistive technology whether it's an info, warning, or error icon.

### Expected Behavior

Ideally, the aria-label should be something like "Error, read annotations, row 1" or "Warning, read annotations, row 1", where the string starts with the icon type of the button being focused.

### Current Behavior

When testing (with `enableKeyboardAccessibility` enabled), VoiceOver on Mac reads the icon as "Read annotations, row 1, button". No information about the icon itself is communicated to screen readers.

### Reproduction Steps

1. Open ace with a language that supports syntax checking (e.g. Javascript in the [kitchen sink](https://ace.c9.io/build/kitchen-sink.html))
2. Cause a syntax error or a warning.
3. See that the `.ace_gutter_annotation` in the DOM has an `aria-label` without the icon type mentioned in it.

### Additional Information/Context

Relevant lines: [src/layer/gutter.js](https://github.com/ajaxorg/ace/blob/d296fd4af374cb7dad41c437611b9da81761b780/src/layer/gutter.js#L445)

### Ace Version / Browser / OS / Keyboard layout

^1.32.6 / Any / Any / Qwerty

**Comment by InspiredGuy** [comment]
Seems like this was resolved by https://github.com/ajaxorg/ace/pull/5524 back then, closing the issue
